system_baseline/models.py

```diff
@@ -98,3 +98,10 @@ class SystemBaselineMappedSystem(db.Model):
         UUID(as_uuid=True), ForeignKey("system_baselines.id"), nullable=False
     )
     system_id = db.Column(UUID(as_uuid=True), nullable=False, index=True)
+
+    @classmethod
+    def delete_by_system_ids(cls, system_ids, account_number):
+        cls.query.filter(cls.system_id.in_(system_ids)).delete(
+            synchronize_session="fetch"
```

Comment:
I'm a bit fuzzy on the db side; what is the synchronize_session="fetch" for here as opposed to like db.session.add or remove or what not and then db.session.commit when you're finished?

Reply:
synchronize_session="fetch" is for syncing objects in sessions with what we just modify (delete) here. See https://docs.sqlalchemy.org/en/14/orm/session_basics.html#orm-expression-update-delete

I think our session should commit as I see it complete.
We used this approach when deleting more records at once - https://github.com/RedHatInsights/system-baseline-backend/blob/master/system_baseline/views/v1.py#L165-L168 and https://github.com/RedHatInsights/historical-system-profiles-backend/blob/master/historical_system_profiles/db_interface.py#L80-L84

Do you have any preference?